manifest.json

```diff
@@ -3,7 +3,7 @@
 
     "name": "DogeCam",
     "description": " A chrome extension to make your video calls Doge-y!",
-    "version": "0.1.3",
+    "version": "0.2.0",
```

Comment:
why 0.1.3 -> 0.2.0?

Reply:
A significant feature/step? We had been fixing issues on features we started with on earlier increments.